clj-refactor.el

```diff
@@ -1887,39 +1907,25 @@ root."
     (cljr--setup-find-symbol-buffer (or symbol-name symbol))
     (cljr--find-symbol (or symbol-name symbol) ns 'cljr--format-and-insert-symbol-occurrence)))
 
-(defun cljr--read-symbol-metadata (occurrences)
-  (->> occurrences
-       (-partition 7)
-       (-map (lambda (symbol-meta)
-               (apply (lambda (line-start line-end col-start col-end name file match)
-                        (list :line-start line-start
-                              :line-end line-end
-                              :col-start col-start
-                              :col-end col-end
-                              :name (first (last (s-split "/" name)))
-                              :file file
-                              :match match))
-                      symbol-meta)))))
-
-(defun cljr--rename-symbol (ns occurrences new-name)
+(defun cljr--rename-occurrences (ns occurrences new-name)
   (save-excursion
     (dolist (symbol-meta occurrences)
       (with-current-buffer
           (find-file-noselect (plist-get symbol-meta :file))
         (goto-char (point-min))
-        (let* ((line-start (plist-get symbol-meta :line-start))
-               (line-e (or (plist-get symbol-meta :line-end) line-start))
-               (line-end line-start)
-               (column-start (1- (or (plist-get symbol-meta :col-start) 1)))
-               (start (progn (forward-line (1- line-start))
-                             (move-to-column column-start)
+        (let* ((line-beg (plist-get symbol-meta :line-beg))
+               (line-e (or (plist-get symbol-meta :line-end) line-beg))
+               (line-end line-beg)
+               (col-beg (1- (or (plist-get symbol-meta :col-beg) 1)))
+               (start (progn (forward-line (1- line-beg))
+                             (move-to-column col-beg)
                              (point)))
-               (column-end (if (= line-start line-e)
-                               (line-end-position)
-                             (or (plist-get symbol-meta :col-end) (line-end-position))))
+               (col-end (if (= line-beg line-e)
+                            (line-end-position)
+                          (or (plist-get symbol-meta :col-end) (line-end-position))))
                (end (progn (goto-char (point-min))
                            (forward-line (1- line-e))
-                           (move-to-column column-end)
+                           (move-to-column col-end)
                            (point)))
                (orig-name (plist-get symbol-meta :name))
```

Comment:
I just tried running this branch (can't get a repl going on the computer at work with this branch due to what looks like `mranderson` problems) and `orig-name` here is fully qualified so the renaming fails.  I haven't touched the code in the middleware creating the name, was it always like this @benedekfazekas?